aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -0,0 +1,391 @@
+              onSubmit={({ value }) => console.log(value)}
+            >
+              {steps[activeIndex].inputs(
+                activeIndex,
+                setActiveIndex,
+                formValues,
+              )}
+            </Form>
+          </Box>
+        </Box>
+      </Box>
+      {open && <CancellationLayer onSetOpen={setOpen} />}
+    </>
+  );
+};
+
+const WizardHeader = ({ activeIndex, activeStep, setActiveIndex, setOpen }) => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Header
+      border={{ side: 'bottom', color: 'border-weak' }}
+      pad="small"
+      fill="horizontal"
+      justify="center"
+    >
+      <Box direction="row" flex>
+        {activeStep > 1 && (
+          <Button
+            label={size !== 'small' ? `Step ${activeStep - 1}` : undefined}
+            icon={<FormPreviousLink color="text-strong" />}
+            onClick={() => setActiveIndex(activeIndex - 1)}
+          />
+        )}
+      </Box>
+      <Box>
+        <Text color="text-strong" weight="bold">
+          Action Title
+        </Text>
+      </Box>
+      <Box direction="row" flex justify="end">
+        <Button
+          label={size !== 'small' ? 'Cancel' : undefined}
+          icon={<FormClose color="text-strong" />}
+          reverse
+          onClick={() => setOpen(true)}
+        />
+      </Box>
+    </Header>
+  );
+};
+
+WizardHeader.propTypes = {
+  activeIndex: PropTypes.number.isRequired,
+  activeStep: PropTypes.number.isRequired,
+  setActiveIndex: PropTypes.func.isRequired,
+  setOpen: PropTypes.func.isRequired,
+};
+
+const CancellationLayer = ({ onSetOpen }) => {
+  return (
+    <Layer
+      position="center"
+      onClickOutside={() => onSetOpen(false)}
+      onEsc={() => onSetOpen(false)}
+    >
+      <Box pad="large" gap="small" width="large">
+        <Box>
+          <Heading color="text-strong" margin="none">
+            Cancel
+          </Heading>
+          <Text color="text-strong">Action Title</Text>
+        </Box>
+        <Text color="text-strong">
+          Cancelling setup will lose all of your progress. Are you sure you want
+          to exit the setup?
+        </Text>
+        <Box
+          as="footer"
+          gap="small"
+          direction="row"
+          align="center"
+          justify="end"
+          pad={{ top: 'medium', bottom: 'small' }}
+        >
+          <Button
+            label="No, continue setup"
+            onClick={() => onSetOpen(false)}
+            secondary
+          />
+          <Button
+            label="Yes, cancel setup"
+            onClick={() => onSetOpen(false)}
+            primary
+          />
+        </Box>
+      </Box>
+    </Layer>
+  );
+};
+
+CancellationLayer.propTypes = {
+  onSetOpen: PropTypes.func.isRequired,
+};
+
+const Error = ({ children, ...rest }) => {
+  return (
+    <Box
+      background="validation-critical"
+      margin={{ bottom: 'medium' }}
+      pad="small"
+      round="4px"
+    >
```

Comment:
I think @britt6612 ended up adding a little animation to the error reveal, so that the error doesn't feel like a slap in the face. Check it out and see if you like it or not.

```suggestion
    <Box
      animation="fadeIn"
      background="validation-critical"
      margin={{ bottom: 'medium' }}
      pad="small"
      round="4px"
    >
```

Reply:
Implemented this so the behaviors match.